examples/cf-stackdriver-example/main.go

```diff
@@ -75,12 +75,15 @@ func ListPushHandler(rw http.ResponseWriter, req *http.Request) {
 	}
 
 	datastoreKey := datastore.IncompleteKey(key, nil)
-	datastoreClient.Put(context.Background(), datastoreKey, newEntry)
+	_, err := datastoreClient.Put(context.Background(), datastoreKey, newEntry)
```

Comment:
It's probably worth forwarding on the request context here to demonstrate proper usage